hack/tree_status.sh

```diff
@@ -14,17 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
-set -o nounset
-set -o pipefail
+set -e
 
-ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
-cd ${ROOT}
-
-echo "Run vndr to update vendor"
-vndr -whitelist github.com/onsi/ginkgo/*
-
-echo "Sort vendor.conf..."
-sort vendor.conf -o vendor.conf
-
-echo "Please commit the change made by this file..."
+STATUS=$(git status --porcelain)
```

Comment:
I think we should also run go mod command before checking git status, e.g. 
```
$(GO) mod tidy && \
		$(GO) mod vendor && \
		$(GO) mod verify
```

Reply:
Okay, added this as well.